doc/source/building/build_with_profiles.rst

```diff
@@ -0,0 +1,80 @@
+.. _building-build-with-profiles:
+
+Building images with profiles
+=============================
+
+KIWI supports so-called profiles inside the XML image description, which
+act as namespaces for additional settings to be applied on top of the
+defaults. For further details see :ref:`xml-description-image-profiles`.
+
+
+Local builds
+------------
+
+Local KIWI builds with a specific profile selected can be executed by
+adding the command line flag `--profile=$PROFILE_NAME` to the KIWI
+invocation, e.g.:
+
+.. code-block:: shell-session
+
+   $ sudo kiwi-ng --type iso system build \
+         --profile=workstation \
+         --description path/to/description/directory \
+         --target-dir /tmp/myimage
+
+Consult the manual page of :file:`kiwi` for further details:
+:ref:`commands-kiwi`.
+
+
+Building in the Open Build Service
```

Comment:
But this section is about building on OBS, so it should be rather called "Building on the Open Build Service"?

Reply:
Maybe I'm splitting hairs here, but... well, but that's my job. :wink:  Ok, let me explain.

From my perspective, you use or communicate with the server via the `osc` tool or with the Web page. That server "contains" or "has" a service. 

So when you want to build a package or image on OBS, you build it on the **server** (a physical thing) but not on the **service** (an abstract thing). That's why I would suggest to use server instead of service.

Does that make sense? :wink:

Reply:
It makes a lot of sense if you put it that way, my only concern with that type of wording is that the average user reads this and thinks: "Huh, what on earth is this open build server? I know the open build service but not the open build server... :thinking: Well, I probably won't be needing that then.".

What I mean is: we always market & mention OBS as the open build **service** and never as the open build **server.**

Reply:
That's a valid concern. :+1: On the other hand, what's the "average user"? See, we are coming back to our target group. :smile:

Ok, one suggestions that maybe avoids it could be the following idea:

    Building with Open Build Service

What do you think about that?

Reply:
I'd add a "the" into the title: `Building with the Open Build Service`, but otherwise that sounds good to me.